From 53b89219201613e4e59218a3efbca88b48f806a5 Mon Sep 17 00:00:00 2001
From: Paresh Bhagat <p-bhagat@ti.com>
Date: Thu, 27 Apr 2023 11:10:29 +0530
Subject: [PATCH 13/15] arm64: dts: add reserved_memory label for CMA regions
 for k3-am625-sk

Add lable fore reserved memory needed to build and apply overlay

Upstream-Status: Pending

Signed-off-by: Matt Ranostay <mranostay@ti.com>
Signed-off-by: Paresh Bhagat <p-bhagat@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi b/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi
index 07763091ab5953..60bf8cd6ca4586 100644
--- a/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-am62x-sk-common.dtsi
@@ -35,7 +35,7 @@ memory@80000000 {
 
 	};
 
-	reserved-memory {
+	reserved_memory: reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
 		ranges;
-- 
2.40.1

